netqasm/runtime/cli.py

```diff
@@ -19,6 +19,7 @@
     post_function_from_path,
 )
 from netqasm.runtime.env import get_example_apps, init_folder, new_folder
+from netqasm.runtime.process_logs import _create_app_instr_logs
```

Comment:
The convention is that functions/names starting with an underscore (`_`) are meant to be used only privately, in their own module. I propose to rename the function to `create_app_instr_logs` (without underscore) so that we are not importing a 'private' function anymore.